package.json

```diff
@@ -17,23 +17,19 @@
     "build": "tsc && rollup -c",
     "prepublishOnly": "npm run build",
     "pretest": "npm run build",
-    "test": "TZ=Asia/Dubai karma start ./test/karma.config.cjs",
+    "test": "web-test-runner",
```

Comment:
I really like how we were able to bake the timezone feature directly into `web-test-runner` which allows for different invocations without having to carry environment variables around.